troi/print_recording.py

```diff
@@ -79,6 +83,8 @@ def _print_recording(self, recording, year=False, listen_count=False, bpm=False,
             print(" %4d" % recording.listenbrainz['listen_count'], end='')
         if self.print_bpm or bpm:
             print(" %3d" % recording.acousticbrainz['bpm'], end='')
+        if self.popularity or popularity:
+            print(" %3d" % recording.musicbrainz['popularity'], end='')
```

Comment:
I checked the codebase and the popularity field is a float so `0.3f` would be better than `%3d`

Reply:
made the changes

Reply:
made the changes

Reply:
Thanks.